app/api/v1/api.rb

```diff
@@ -4,5 +4,6 @@ class Api < Grape::API
 
     mount Auth
     mount Healthcheck
+    mount Conversions::ProjectsController
```

Comment:
I would like this to be named a just a resource, so `conversions` and then switched so projects is first, giving us a path of `api/v1/projects/conversions` and later `api/v1/projects/transfers` - it might just be me, but I am more comforatble with that, I think of them all as types of 'projects', we also model them this way.